drivers/hmis/app/graphql/types/forms/form_identifier.rb

```diff
@@ -0,0 +1,45 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class Forms::FormIdentifier < Types::BaseObject
+    skip_activity_log
+    description 'FormIdentifier'
+
+    available_filter_options do
+      arg :search_term, String
+      # ADD: role
+      # ADD: status
+    end
+
+    # TODO: add summary information so the UI can resolve top-level detail such as 'active in 15 projects'
+
+    field :id, String, null: false
+    field :identifier, String, null: false
+    field :role, Types::Forms::Enums::FormRole, null: false
+    field :title, String, null: false
+
+    field :current_version, Types::Forms::FormDefinition, null: false
+    field :draft, Types::Forms::FormDefinition, null: true
```

Comment:
If a form hasn't been published yet (so there is only a draft version), how does it resolve? I see the current_version is non-nullable.

Reply:
Hm, right now `current_versions` returns a draft version if no published version exists. Maybe this is too confusing. I guess the reason I thought to do this is because we basically want a query that returns one FormIdentifier per row that we want to display in the form table, and we want to display both published and draft-only forms there.

Reply:
as discussed, lets rename to published_version and make nullable